client/globals.d.ts

```diff
@@ -153,11 +153,21 @@ interface ResizeObserverCallback {
     (entries: ResizeObserverEntry[], observer: ResizeObserver): void;
 }
 
-export interface ILineConfig {
+export interface ILineConfigStandard {
     fieldId: string;
     position?: 'start' | 'end';
+    fieldOptions: any; // type of options will be different for each field type
 }
 
+export interface ILineConfigVocabulary extends ILineConfigStandard {
+    fieldId: 'vocabulary';
+    fieldOptions: {
+        vocabularyId: string;
```

Comment:
might be useful to be able to hide vocabulary name and only show selected values to save some space in the list

Reply:
I don't think it's a good idea - it wouldn't be possible to differentiate items from different vocabularies. Maybe it'd be fine if only one vocabulary is configured.